mobile/src/calendar/days-counters/days-counter-selected-day.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import { View, Text } from 'react-native';
+import { daysCounterSelectedDayStyles } from '../styles';
+
+interface DaysCounterSelectedDayProps {
+    day: string;
+    month: string;
+}
+
+export  class DaysCounterSelectedDay extends Component<DaysCounterSelectedDayProps> {
```

Comment:
why `DaysCounter`? If it's a circle in the middle, call it SelectedDay?

Reply:
fixed